bom/application/pom.xml

```diff
@@ -433,26 +432,20 @@
                 <version>${opensearch-testcontainers.version}</version>
             </dependency>
             <!-- OpenTelemetry components, imported as a BOM -->
-            <dependency>
-                <groupId>io.opentelemetry</groupId>
-                <artifactId>opentelemetry-bom</artifactId>
-                <version>${opentelemetry.version}</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-            <dependency>
-                <groupId>io.opentelemetry</groupId>
-                <artifactId>opentelemetry-bom-alpha</artifactId>
-                <version>${opentelemetry.version}-alpha</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
+            <!-- opentelemetry-instrumentation-bom-alpha includes the main and alpha-bom inside.-->
+            <!-- Also includes stable semantic conventions-->
             <dependency>
                 <groupId>io.opentelemetry.instrumentation</groupId>
                 <artifactId>opentelemetry-instrumentation-bom-alpha</artifactId>
-                <version>${opentelemetry-alpha.version}</version>
+                <version>${opentelemetry-instrumentation.version}</version>
                 <type>pom</type>
                 <scope>import</scope>
+                <exclusions>
+                    <exclusion>
+                        <groupId>io.opentelemetry.javaagent</groupId>
+                        <artifactId>*</artifactId> <!-- Exclude all artifacts in this group -->
+                    </exclusion>
+                </exclusions>
```

Comment:
@aloubyansky will it help the Platform BOM generator?

Reply:
I don't think it will have any effect

Reply:
Any suggestion on what would work?

Reply:
I added a comment below - configure custom filtering when flattening